### ISSUE TITLE
sample demultiplexing Parse technologies

### ISSUE THREAD

**Issue by estebanelias** [drneavin/Demultiplexing_Doublet_Detecting_Docs#52]
Hi. This is a very useful tool. I'm wondering if this tool can be applied to Parse Biosciences scRNAseq technology. Are you thinking about doing a tutorial about this? Thank you very much.

**Comment by drneavin** [comment]
Hi Esteban,

Yes, we have applied Demuxafy to both Parse and Scalebio with good success. I'm not planning on doing a tutorial for this as the functions are much the same. When I tested it, the UMI tag was UB and the barcode tag was CB but I recommend taking a look at one of your bams to verify that those are the tags being used. You can also find out the predicted doublet rate using our [doublet estimator tool](https://demultiplexing-doublet-detecting-docs.readthedocs.io/en/latest/Calculator_final_version.html). But I'll note that the suggested software combinations are still based on our 10x simulations.

Let me know if you have further questions about demultiplexing with Parse.

**Comment by estebanelias** [comment]
I really appreciate your response. I'll try this with Parse data and I'll let you know if I have more questions. One silly question: do you think that comercial mice can be demultiplexed with success (for example if we used mice same C57 from Jackson).

**Comment by drneavin** [comment]
Yes, please report back. Will be good to hear your experience with Demuxafy applied to Parse data. 

I think demultiplexing mice will depend on if there is enough genetic diversity between the mice. If they're all the same strain of mice, I don't think you'll have much success since there should be relatively little genetic diversity between them (based on my understanding) but if they're different strains or if you know there are genetic locations that are different, you might be able to
